packages/react/src/primitives/utils/testUtils.ts

```diff
@@ -0,0 +1,16 @@
+export const errorMessageWrapper = (fn: () => void, message: string) => {
```

Comment:
This function allows us to render a customer error message for an expect that throws.